src/dapps/nft.ts

```diff
@@ -84,7 +84,7 @@ export type NFTFilters = {
   wearableGenders?: WearableGender[]
   emoteCategory?: EmoteCategory
   emoteGenders?: WearableGender[]
-  emotePlayMode?: EmotePlayMode
+  emotePlayModes?: EmotePlayMode[]
```

Comment:
This would be a breaking change, I'd suggest adding it as a different prop and keep the individual one as well